### PR TITLE
Improve accessibility

### PR DESCRIPTION
**Pull request by michalczaplinski**
- Added `aria-controls` labels to the buttons that toggle the trailer player.
- Added the `<label>` element to the search and wrap in in a `<form>`.usingaccessible thanks to:
	- Added `aria-selected` & `role` attributes on the tabs.
	- Added `aria-hidden` attributes to the tab panels.
	- Added `aria-labelledby` to connect the tabs with the tab panels.

### CODE REVIEW

**Comment by SantosGuillamot** [comment]
Thanks for the pull request 🙂 I still have to take a deeper look, but it looks great so far! A couple of comments:

* In the movie tabs block, I believe we could add some `aria` attributes to the Images/Videos buttons and the corresponding `<div>`. Something similar to [this example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example).
* In the same way we are adding `aria-controls` to the buttons that toggle the trailer player, should we do something similar for the Like Icon and Like button blocks, that modify the Likes number?

**Comment by michalczaplinski** [comment]
Hmm, looks like the `aria-expanded` attributes should only be applied to elements that can be expanded/collapsed like a treeview or an expandable menu so this doesn't really apply to any blocks that we have in the demo.

File: src/blocks/interactive/movie-tabs/render.php [hunk 1]
```diff
@@ -22,51 +22,71 @@
...
+			</button>
 		</li>
 	</ul>
 
-	<div data-wp-show="selectors.wpmovies.isImagesTab">
+	<div 
+		role="tabpanel" 
+		data-wp-show="selectors.wpmovies.isImagesTab" 
+		aria-hidden="selectors.wpmovies.isVideosTab" 
+		aria-labelledby="wpmovies-images-tab"
```
Comment (SantosGuillamot) [inline src/blocks/interactive/movie-tabs/render.php:56]: I guess we have to use `data-wp-bind.aria-hidden`, right?

**Comment by SantosGuillamot** [comment]
> As far as I understand (tho my understanding is admittedly not great) the aria-controls should be used for elements that are shown and hidden according to e.g. [this article](https://adrianroselli.com/2020/05/disclosure-widgets.html). So we don't have to add them for the Like Icon and Like Button blocks.

Okay, I wasn't sure 🙂 . I read the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls), and as it says _"The global aria-controls property identifies the element (or elements) whose contents or presence are controlled by the element on which this attribute is set."_ I understood it could apply.

I added a couple of comments, but for the rest, it looks good to me (although I don't know that much about accessibility) 👍

**Review by SantosGuillamot — APPROVED** [review]

File: src/blocks/interactive/movie-tabs/render.php (hunk 1 above)
Comment (michalczaplinski) [reply src/blocks/interactive/movie-tabs/render.php:56]: indeed!

File: src/blocks/interactive/movie-tabs/render.php [hunk 2]
```diff
@@ -22,51 +22,71 @@
...
 			}
 			?>
 		</div>
 	</div>
 
-	<div data-wp-show="selectors.wpmovies.isVideosTab">
+	<div 
+		role="tabpanel" 
+		data-wp-show="selectors.wpmovies.isVideosTab" 
+		aria-hidden="selectors.wpmovies.isImagesTab"
```
[the remark this answers is not in the document]
Comment (michalczaplinski) [reply src/blocks/interactive/movie-tabs/render.php:73]: yep!

**Comment by michalczaplinski** [comment]
Nice, thanks for the review, I ll merge it now! 🙂

**Pull request merged**